src/wizard/javascript/angular/with-error-monitoring-and-performance.md

```diff
@@ -0,0 +1,105 @@
+
+Add the Sentry SDK as a dependency using `yarn` or `npm`:
+
+```bash
+# Using yarn (Angular 12+)
+yarn add @sentry/angular-ivy
+# Using yarn (Angular 10 and 11)
+yarn add @sentry/angular
+
+# Using npm (Angular 12+)
+npm install --save @sentry/angular-ivy
+# Using npm (Angular 10 and 11)
+npm install --save @sentry/angular
+```
+
+## Configure
+You should `init` the Sentry browser SDK in your `main.ts` file as soon as possible during application load up, before initializing Angular:
+
+```javascript
+import { enableProdMode } from "@angular/core";
+import { platformBrowserDynamic } from "@angular/platform-browser-dynamic";
+// import * as Sentry from "@sentry/angular" // for Angular 10/11 instead
+import * as Sentry from "@sentry/angular-ivy";
+
+import { AppModule } from "./app/app.module";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [
+    new Sentry.BrowserTracing({
+      tracePropagationTargets: ["localhost", "my-site-url.com", /^\//],
+      routingInstrumentation: Sentry.routingInstrumentation,
+    }),
+  ],
+  // Performance Monitoring
+  tracesSampleRate: 1.0, // Capture 100% of the transactions, reduce in production!
+});
+
+enableProdMode();
+platformBrowserDynamic()
+  .bootstrapModule(AppModule)
+  .then(success => console.log(`Bootstrap success`))
+  .catch(err => console.error(err));
+```
+
+### ErrorHandler and Tracer
+
+The Sentry Angular SDK exports a function to instantiate `ErrorHandler` provider that will automatically send JavaScript errors captured by the Angular's error handler.
+
+```javascript
+import { APP_INITIALIZER, ErrorHandler, NgModule } from "@angular/core";
+import { Router } from "@angular/router";
+// import * as Sentry from "@sentry/angular" // for Angular 10/11 instead
+import * as Sentry from "@sentry/angular-ivy";
+
+@NgModule({
+  // ...
+  providers: [
+    {
+      provide: ErrorHandler,
+      useValue: Sentry.createErrorHandler({
+        showDialog: true,
+      }),
+    },
+    {
+      provide: Sentry.TraceService,
+      deps: [Router],
+    },
+    {
+      provide: APP_INITIALIZER,
+      useFactory: () => () => {},
+      deps: [Sentry.TraceService],
+      multi: true,
+    },
+  ],
+  // ...
+})
+export class AppModule {}
+```
+
+## Verify
+This snippet contains an intentional error and can be used as a test to make sure that everything's working as expected.
+
+```javascript
+myUndefinedFunction();
+```
+
+---
+## Next Steps
+- [Source Maps](https://docs.sentry.io/platforms/javascript/guides/angular/sourcemaps/): Learn how to enable readable stack traces in your Sentry errors.
+- [Angular Features](https://docs.sentry.io/platforms/javascript/guides/angular/features/): Learn about our first class integration with the Angular framework.
+- [Performance Monitoring](https://docs.sentry.io/platforms/javascript/guides/angular/performance/): Track down transactions to connect the dots between 10-second page loads and poor-performing API calls or slow database queries.
```

Comment:
I would still leave this here so that users can still read more about performance. There additional features to enable (component tracking, database instrumentation etc) so it's a good CTA to have.

Reply:
but users will think that this is a next step... 'next steps' is different from 'read more'. @Jesse-Box what is your thought on this?

Reply:
So, I initially created this footer section as a cheap way to onboard users to important features or products. I agree with @priscilawebdev, that since we have this interface to select products to set up, Including Performance Monitoring as a bullet item doesn't make sense anymore.